pax-jdbc-config/src/main/java/org/ops4j/pax/jdbc/config/impl/Decryptor.java

```diff
@@ -52,7 +53,7 @@ public Decryptor(ServiceTracker encryptorServiceTracker) {
                     decryptedConfig.put(key, plainText);
                 }
             } else {
-                decryptedConfig.put(key, value);
+                decryptedConfig.put(key,String.valueOf(value));
```

Comment:
As the only strings can be encrypted, why didn't you use the String.valueOf on line 39, so the rest could be still the same?
Second, at OPS4j, we actually just have one mandatory constraint for accepting PullRequest (and especially for bugfixes): Do you have a unit test for this fix? Otherwise, how is it possible to verify that this actually fixes the problem, and even more important. Otherwise it's not possible to detect a regression in future changes.